scripts/fix_embargo_approved_logs.py

```diff
@@ -0,0 +1,47 @@
+"""
+Update existing "embargo_approved_no_user" logs to link to registered project instead
+of the registration.
+"""
+import logging
+import sys
+
+from modularodm import Q
+
+from framework.transactions.context import TokuTransaction
+
+from website.models import Node, NodeLog
+from website.app import init_app
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+
+
+def main():
+    targets = get_targets()
+    fix_embargo_approved_logs(targets)
+
+
+def get_targets():
+    return NodeLog.find(Q('action', 'eq', NodeLog.EMBARGO_APPROVED) & Q('params.user', 'eq', None))
+
+
+def fix_embargo_approved_logs(targets):
```

Comment:
Ah, I didn't catch the logging.debug statement a few lines down. You should use `logger.info` instead so that the logs are written to a file (which is what add_file_logger does).